pages/_document.tsx

```diff
@@ -17,6 +18,18 @@ export default function Document() {
       </style>
       <Head>
         <meta name="description" content="Research Catalog | NYPL" />
+        <script async src={appConfig.adobeEmbedUrl[appConfig.environment]} />
```

Comment:
Should this use the nextjs `NextScript` component?

Reply:
Good call, I replaced this with a <Script> tag.